README.md

```diff
@@ -95,12 +95,20 @@ Applications use the CSync class to create a connection to a specific CSync serv
 final CSync csync = new CSync.builder()
     .host("localhost")
     .port(6005)
-    .provider("demo")
-    .token("demoToken")
     .build();
 ```
 
-Note: Update the `host` and `port` to your specific csync server instance. Also, `provider` and `token` can be updated to allow other providers to authenticate the csync session.
+Note: Update the `host` and `port` to your specific csync server instance.
+
+## Authenticating
+```
+csync.authenticate("demo", "demoToken")
```

Comment:
This should be written out more. Even a simple example like

if(didSucceed) {
//We successfully logged in
}
else {
ex.printStackTrace()
}